src/renderer/views/AddProject/AddProject.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { styled } from '@mui/system';
+import { Typography } from '@mui/material';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  decreaseStep,
+  increaseStep,
+} from '../../../shared/redux/slices/addPublicationSlice';
+import { RootState } from '../../../shared/redux/rootReducer';
+
+const StyledBackground = styled('div')(({ theme }) => ({
+  width: '100%',
+  height: '100%',
+  background: theme.palette.background,
+}));
+
+const StyledContainer = styled('div')(({ theme }) => ({
+  marginLeft: 'auto',
+  marginRight: 'auto',
+  display: 'flex',
+  flexDirection: 'column',
+  width: '50%',
+  background: theme.palette.background,
+  paddingTop: theme.spacing(2),
+}));
+
+const StepContainer = styled('div')(({ theme }) => ({
+  marginTop: theme.spacing(10),
+  marginBottom: theme.spacing(8),
+}));
+
+const StyledButtonBlack = styled('button')(({ theme }) => ({
+  width: '50%',
+  height: '60px',
+  border: '1px solid',
+  borderColor: theme.palette.text.primary,
+  outline: 'none',
+  color: theme.palette.text.primary,
+  backgroundColor: 'transparent',
+  fontSize: '15px',
+  fontWeight: 'bold',
+  '&:hover': {
+    cursor: 'pointer',
+  },
+}));
+
+const StyledButtonGreen = styled('button')(({ theme }) => ({
+  width: '50%',
+  height: '60px',
+  border: 'none',
+  outline: 'none',
+  background: theme.palette.green.main,
+  fontSize: '15px',
+  fontWeight: 'bold',
+  '&:hover': {
+    cursor: 'pointer',
+  },
+}));
```

Comment:
Hmm, why aren't we using the project's `Button` component here but instead styling new ones?

Also, In case of _css-in-js_ styling it is always good to extract them to another file like `style.ts` as well.